iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -276,6 +276,10 @@ public void configureJobConf(TableDesc tableDesc, JobConf jobConf) {
       String tables = jobConf.get(InputFormatConfig.OUTPUT_TABLES);
       tables = tables == null ? tableName : tables + TABLE_NAME_SEPARATOR + tableName;
       jobConf.set(InputFormatConfig.OUTPUT_TABLES, tables);
+      // todo remove
```

Comment:
is this leftover?

Reply:
Thanks. It was added when debugging code.
Removed it.